Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Allocator/Jobs/ScheduledRides/ScheduledRideNotificationsToDriver.hs

```diff
@@ -139,19 +101,94 @@ sendScheduledRideNotificationsToDriver Job {id, jobInfo} = withLogTag ("JobId-"
           formattedBody = T.replace "{#pickupAddress#}" fullAddress $ T.replace "{#isRentalOrIntercity#}" isRentalOrIntercity $ T.replace "{#driverPartnerName#}" driverPartnerName body
       (formattedTitle, formattedBody)
 
-    buildCallStatus callStatusId exotelResponse booking = do
+sendTagActionNotification ::
```

Comment:
I think this function and next one `sendCommuncationToDriver` should be moved to separate module, because it used for namma-tag action notifications